client/cli/go/glide.yaml

```diff
@@ -0,0 +1,15 @@
+package: github.com/heketi/heketi/client/cli/go
+import:
+- package: github.com/heketi/heketi
```

Comment:
Ack. Will remove this package. I have been having other issues with the Makefile. The client heketi package has its own dependencies; it is trying to create a vendor file, glide.yaml and a lock file. Any idea how to tackle that issue.